src/components/ButtonClientMobile.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-env browser */
```

Comment:
Honestly, i don't know. ESLint doc said this but I'm open to suggestions. Do you guys think `/* global localStorage */` is more relevant?